specification.markdown

```diff
@@ -653,16 +626,10 @@ description: The minimum `DocType` version an `EBML Reader` has to support to re
 
 name: CRC-32
 
-level: 1
-
-global: true
+path: `/*/CRC-32(0,1)`
```

Comment:
Ok, ignore the prior CRC-32 comment, I see that `*` implies the any position. I'll move my comments to the section on the meaning of `*`.